version.go

```diff
@@ -1,3 +1,3 @@
 package main
 
-const VERSION = "2.7.0"
+const VERSION = "2.8.0"
```

Comment:
I agree but I prefer to do the release separately. I like to bump the version and update the CHANGELOG.md at the same time (and then tag that commit).